docs/how_to_guides/how_to_use_inherent_reactions.rst

```diff
@@ -289,15 +288,21 @@ code below.
     # NOTE: What is different here is now we state that there are no
     #       equilibrium reactions in this unit model because we defined
     #       those reactions as inherent.
-    model.fs.unit = EquilibriumReactor(default={
-                "property_package": model.fs.thermo_params,
-                "reaction_package": model.fs.rxn_params,
-                "has_rate_reactions": False,
-                "has_equilibrium_reactions": False,
-                "has_heat_transfer": False,
-                "has_heat_of_reaction": False,
-                "has_pressure_change": False})
+    model.fs.unit = EquilibriumReactor(property_package=model.fs.thermo_params,
+                                       reaction_package=model.fs.rxn_params,
+                                       has_rate_reactions=False,
+                                       has_equilibrium_reactions=False,
+                                       has_heat_transfer=False,
+                                       has_heat_of_reaction=False,
+                                       has_pressure_change=False)
 
     # At this point, you can 'fix' your inlet/outlet state conditions,
     #     setup scaling factors, initialize the model, then solve the model
     #     just as you would with any other IDAES flowsheet
+
+.. testoutput::
+   :hide:
+   :options: +ELLIPSIS
+
+   WARNING: DEPRECATED: The default argument for the ProcessBlock class is
+       ...
```

Comment:
Same re: need to remove once deprecation warnings are addressed.

Reply:
Yes, this will be revised once the deprecation warnings are fixed.